### ISSUE TITLE
Not all STIGs are importing

### ISSUE THREAD

**Issue by jeremytourville** [NUWCDIVNPT/stig-manager#834]
I have downloaded the STIG compilation library [from https://dl.cyber.mil/stigs/zip/CUI_SRG-STIG_Library_2022_10v2.zip](https://github.com/NUWCDIVNPT/stig-manager/issues/from%20https:/dl.cyber.mil/stigs/zip/CUI_SRG-STIG_Library_2022_10v2.zip)
Please see the error I receive when attempting to utilize the RHEL 8 STIG via an import process.

I have also included RHEL 7 as example of what happens when the STIG imports properly/as expected.

```
------------------------------------
Extracting member U_RHEL_7_V3R9_STIG.zip
Processing member U_RHEL_7_V3R9_STIG.zip
U_RHEL_7_V3R9_Manual_STIG/U_RHEL_7_STIG_V3R9_Manual-xccdf.xml
{
  "dmlObject": "Built in 0s  3.174765ms",
  "stig": "0 in 0s  0.468338ms",
  "revision": "0 in 0s  0.42711ms",
  "group": "237 in 0s  2.634444ms",
  "rule": "0 in 0s  7.956784ms",
  "check": "0 in 0s  9.711906ms",
  "fix": "0 in 0s  3.518155ms",
  "revGroupMap": "0 in 0s  8.661282ms",
  "revGroupRuleMap": "0 in 0s  6.076217ms",
  "revGroupRuleCheckMap": "0 in 0s  7.706103ms",
  "revGroupRuleFixMap": "0 in 0s  8.598006ms",
  "ruleCciMap": "0 in 0s  10.674347ms",
  "currentRev": "1 in 0s  4.439078ms",
  "currentGroupRule": "237 in 0s  8.588897ms",
  "stats": "237 in 0s  0.971923ms",
  "totalTime": "Completed in 0s  84.53383ms"
}
------------------------------------
Extracting member U_RHEL_8_V1R8_STIG.zip
Processing member U_RHEL_8_V1R8_STIG.zip
U_RHEL_8_V1R8_Manual_STIG/U_RHEL_8_STIG_V1R8_Manual-xccdf.xml
Error processing ZIP: JSON.parse: unexpected character at line 1 column 1 of the JSON data
```

Any suggestions for troubleshooting?  I reached out to DISA and they say "the RHEL 8 STIG loads into STIG Viewer, which is it’s intended use case."

**Comment by csmig** [comment]
So far I'm not able to replicate this using the CUI Library from the link provided. It would be helpful to know whether you are able to import the individual RHEL_8 ZIP. Perhaps you could extract the RHEL_8 ZIP from the Library and try importing it individually and report back. Thanks

**Comment by jeremytourville** [comment]
See if this link works for you.  [DoD Cyber Exchange  - Pulblic ]( https://public.cyber.mil/stigs/compilations/l)

That should get you the STIG dated 16 Nov 2022 with the title:  **Compilation - SRG-STIG Library** 

I have tried extracting the RHEL 8 zip from the library.  It makes no difference, the result is still the same,

**Comment by cd-rite** [comment]
Hi @jeremytourville Are you using a reverse proxy in your deployment? If so, I suspect this issue may be related to a max upload size being enforced.  I am able to recreate an issue very close to yours using NGINX, which by default enforces a 1MB limit on uploads. That Linux STIG is pretty much the biggest one available, and is over 1MB in size, so it returns an error page instead of the JSON we expect. 
I added  `client_max_body_size        100M;` to the server block of my nginx.conf file, and after a restart the problem was resolved. 

We have an issue ([#620 ](https://github.com/NUWCDIVNPT/stig-manager/issues/620)) to present more info in our error messages, which may help you diagnose these issues more easily.  

Let me know if that change works for you! Thanks!
